Source/SwiftLintFramework/ASTRule.swift

```diff
@@ -16,3 +16,10 @@ protocol ASTRule: Rule {
         kind: SwiftDeclarationKind,
         dictionary: XPCDictionary) -> [StyleViolation]
 }
+
+public protocol RuleExample {
+    var ruleName: String { get }
```

Comment:
These first two members (ruleName and ruleDescription) should probably go in `Rule` instead.